### ISSUE TITLE
Swipe on Navbar

### ISSUE THREAD

**Issue by cemkozinoglu** [tgyhlsb/THTinderNavigationController#6]
Hey, this is awesome controller. Didnt had too much time to dig into the code but in example when I run in my phone/simulator, the touch gestures are only captured outside the navbar. Tinder however you can swipe on navbar as well. Is it by design on your controller or a bug? Thanks!

**Comment by carayolthomas** [comment]
Heyy, I wanted to do the same thing so I take a look into the code and it's not implemented. 
Did you manage to do something?

**Comment by cemkozinoglu** [comment]
@carayolthomas yup ended up implementing it, was annoying as hell :) but works. Remind me tomorrow I can send you the code.

**Comment by carayolthomas** [comment]
@cemkozinoglu Nice ;) It would be a pleasure if you could send me the code. Cheers :)

**Comment by lar248** [comment]
hey i'm also trying to get the navigation bar swipe functionality. any chance you could send me that code as well? Thank you!

**Comment by lar248** [comment]
@cemkozinoglu ^

**Comment by ssuchanowski** [comment]
I've added that on my fork over here -> https://github.com/ssuchanowski/THTinderNavigationController
